en-US/firefox/best-browser.lang

```diff
@@ -0,0 +1,116 @@
+
+
+;A browser that minds its business.
+A browser that minds its business.
+
+
+;Privacy on the web is a hot button issue. If privacy is number one on your list of priorities, you want to look for a browser that takes that seriously. When choosing the best private browser for you, look at the tracking policy and how a browser handles your data. These seem like technical questions, but they’re the reason some browsers are more private than others.
+Privacy on the web is a hot button issue. If privacy is number one on your list of priorities, you want to look for a browser that takes that seriously. When choosing the best private browser for you, look at the tracking policy and how a browser handles your data. These seem like technical questions, but they’re the reason some browsers are more private than others.
+
+
+;Trackers are all those annoying “cookies” messages you get on airline sites. These third-party trackers know where you click and can be used to analyze your behavior. A private browser should give users the option to turn off third-party trackers, but ideally, turn them off by default.
+Trackers are all those annoying “cookies” messages you get on airline sites. These third-party trackers know where you click and can be used to analyze your behavior. A private browser should give users the option to turn off third-party trackers, but ideally, turn them off by default.
+
+
+;Another way to stop trackers from tracking is using private mode to browse. Any browser that claims to be private should offer browsing in private mode.
+Another way to stop trackers from tracking is using private mode to browse. Any browser that claims to be private should offer browsing in private mode.
+
+
+;One easy way to check is to visit a browser’s content setting page and privacy policy. The privacy webpage should outline if your data is shared and why. It’s why the <a href="%(privacy)s">Firefox privacy notice</a> is easy to read and easy to find.
+One easy way to check is to visit a browser’s content setting page and privacy policy. The privacy webpage should outline if your data is shared and why. It’s why the <a href="%(privacy)s">Firefox privacy notice</a> is easy to read and easy to find.
+
+
+;Choosing the best browser for you is a lot like choosing a home. You want to explore your options, do some research and make a decision based on what’s important to you.
+Choosing the best browser for you is a lot like choosing a home. You want to explore your options, do some research and make a decision based on what’s important to you.
+
+
+;At <a href="%(firefox)s"></a>Firefox, we’ve worked hard to build a browser that is twice as fast as before and gives users more control over their online life.
```

Comment:
The link should wrap around "Firefox", spotted in https://github.com/mozilla/bedrock/pull/7334#discussion_r298378971

Reply:
Good call.